### ISSUE TITLE
Receive confirmation email

### ISSUE THREAD

**Issue by Dianadec** [bcgov/reserve-rec-public#56]
#### Description:

Receive a confirmation email that is trustworthy +understandable + specific.

#### Acceptance Criteria:
[Note: Use 'Given/When/Then' format if it makes sense to. Otherwise, a simple checklist that can be tested.]

#### Development Checklist:

- [ ] ...
- [ ] ...
- [ ] ...

#### Dependencies

- Blocked by
- Blocking

**Relevant documentation as reference**

**Definition of Ready**

- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if applicable)
- [ ] Design / Solution is accepted by Product Owner (if applicable)
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

**Definition of Done** 
- In progress:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] UI meets accessibility requirements
  - [ ] Unit tests are written
  - [ ] Work is traceable in GitHub
  - [ ] PR linked to ticket number
  - [ ] If needed/required - Dev adds flag/label to highlight any migration steps necessary prior to PROD deployment
- Code review:
  - [ ] Code is peer reviewed and has passed CI/CD tests
- QA:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Code is potentially shippable to the production environment
  - [ ] Functional features have been tested and passed by QA
  - [ ] UI components tested by designer
  - [ ] Code is deployed to PROD when moved to 'done' column (unless requested otherwise by PO)
- PO Review:
  - [ ] Acceptance criteria are tested (Functionality meets the acceptance criteria defined in the ticket)
  - [ ] Reviewed and approved by Product Owner

#### Notes:
- 
-

**Comment by manuji** [comment]
Tested on TEST: **Pending**
* When a pass is booked, the SMS was received but not the booking confirmation email. 
* Tried with two accounts and both accounts had email addresses whitelisted. 
<img width="1125" height="880" alt="Image" src="https://github.com/user-attachments/assets/ee1f3ce5-4c21-409d-8dd3-17e7d3a8e185" />

**Comment by marklise** [comment]
Turns out we needed a TEST deployment of the email templates. You can resume testing again.

**Comment by manuji** [comment]
Tested on TEST: **Pending**
* Emails are still not received. 
* As per @marklise it is possible that the templates are NOT uploaded to TEST.

**Comment by marklise** [comment]
Email Q was missing from env. Testing can resume.

**Comment by manuji** [comment]
Tested on TEST: **Pending**
* Re-tested and I still don't get the booking confirmation emails on TEST environment. 
* Moving back to Todo column.

**Comment by manuji** [comment]
re-tested and still not getting emails. (got the SMS for the booking)

**Comment by manuji** [comment]
Tested on TEST: **Failed**

1. The QR code is broken and not displayed on the email confirmations. 
2. The button to download the QA code does not function. 
3. The bcparks logo is incorrectly displayed on white background. 
4. Email header says "Passes" even when one pass has been booked. 
5. Incorrect copy displayed on the email template above the "View your booking" button. 
6. Display the date and product name instead of the activity type below the park name in gray box
7. Missing arrival and Departure information.
8.  Display the date along with the pass type or product name. 
9. Copy for the Cancellation and refunds mention refund and fees when there is no fees associated to the passes. 
10. Missing copy under the Before you go section. 
11. UI of email template does not match the figma designs. 

<img width="449" height="1250" alt="Image" src="https://github.com/user-attachments/assets/215263d2-dcc8-4846-b446-c5ee2130f89b" />

<img width="664" height="667" alt="Image" src="https://github.com/user-attachments/assets/266a3663-ed1f-4da6-a77f-31861060caae" />

**Comment by marklise** [comment]
PR for the QA fixes: bcgov/reserve-rec-api#417 (auto-merge enabled; will deploy to TEST on the next release tag, which syncs the updated email templates to the TEST bucket).

**Fixed in this PR (QA items):** #1 QR now renders (delivered as an inline CID attachment — email clients strip `data:` URIs), #2 broken download button removed, #4 pass/passes pluralization, #5 intro copy, #6 gray-box subtitle (now "Day-use pass"), #7 missing Arrival/Departure (data bug), #8 pass type + date, #9 cancellation section removed, #10 Customer Service box replaced with the `parkinfo@gov.bc.ca` disclaimer, #11 layout reworked to match the design.

**Needs a design asset — QA #3 (logo):** the confirmation header in the design shows the **colour** BC Parks logo on a white background, but there is **no colour/positive BC Parks logo** anywhere in the three repos — only reversed/white variants — and there's no stable hosted URL for one. As an interim the email ships a white "BC Parks" wordmark on a brand-blue band (renders reliably in all clients, unlike the reversed SVG which Gmail strips). To match the design exactly we need the official colour horizontal BC Parks PNG from the designer; can raise a separate design ticket for that asset.

**Comment by manuji** [comment]
Tested on TEST: **Passed**
* Verified that the issues that have been fixed are working fine. 
* Passing to PO review.

**Comment by LindsayMacfarlane** [comment]
Approved. 

Issue already noted and there is a follow-up ticket > what I receive looks quite different then designs through my government account.

**Comment by manuji** [comment]
This ticket is already deployed to PROD.
